helpdesk_mgmt_ticket_stage_auto/models/helpdesk_ticket_stage.py

```diff
@@ -0,0 +1,67 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+from odoo import fields, models, api, _
+from odoo.exceptions import ValidationError
+import datetime
+
+
+class HelpdeskTicketStage(models.Model):
+    _inherit = "helpdesk.ticket.stage"
+
+    auto_stage = fields.Boolean(default=False)
+    destination_stage = fields.Many2one("helpdesk.ticket.stage")
+    inactivity_time = fields.Integer(
+        string="Inactivity time for stage change(Days)",
+        default=1
```

Comment:
Maybe the default value should be higher (like 30, or 60 days), but I'm not sure.